### ISSUE TITLE
Unable to build with gulp and node-sass

### ISSUE THREAD

**Issue by borys-p** [vadikom/smartmenus#65]
Hi,

I can't build a theme file, because there's a filename conflict with node-sass. When I'm trying to do:

```@import "sm-simple/sm-simple";```

I end up with:

```
Error: It's not clear which file to import for '@import "sm-simple/sm-simple"'.
       Candidates:
         sm-simple/_sm-simple.scss
         sm-simple/sm-simple.scss
         sm-simple/sm-simple.css
       Please delete or rename all but one of these files.
        on line 34 of scss/style.scss
```

Unfortunately even using a full file name doesn't work:

```@import "sm-simple/sm-simple.scss";```

```
Error: It's not clear which file to import for '@import "sm-simple/sm-simple.scss"'.
       Candidates:
         sm-simple/sm-simple.scss
         sm-simple/_sm-simple.scss
       Please delete or rename all but one of these files.
        on line 34 of scss/style.scss
```

Probably grunt does it differently and it works there. Any chance to rename those files?

**Comment by vadikom** [comment]
Ah, seems to be an issue with newer node-sass releases. I will consider renaming the partial "_sm-simple.scss" to something else. But for now, you have a few options:

A) Use:

`@import "sm-simple/sm-simple.scss";`

and either:

1) remane "_sm-simple.scss" and update the import line in "sm-simple.scss" respectively

or

2) append the contents of "_sm-simple.scss" to "sm-simple.scss" and delete the partial "_sm-simple.scss".

B) Rename "sm-simple.scss" to (e.g. "sm-simple-theme.scss") and then import it.

Here is what I found on the subject:

https://github.com/sass/node-sass/issues/1222
http://stackoverflow.com/questions/33612388/how-to-address-node-sass-regression-its-not-clear-which-file-to-import-for

**Comment by borys-p** [comment]
Yes, for now I have made a copy of the SCSS files with changed their names, but it would be much easier to use the npm library directly.

**Comment by vadikom** [comment]
Yes, thanks for your input! I will make sure this is fixed in the next release.

**Comment by vadikom** [comment]
This should be fixed now in 1.1.0.

**Comment by mcropper** [comment]
I'm still getting this error.
